fluidkeys/main.go

```diff
@@ -394,23 +401,28 @@ func promptYesOrNo(message string, defaultResult bool) bool {
 	}
 }
 
-func promptForInputWithPipes(prompt string, reader *bufio.Reader) string {
+func promptForInputWithPipes(prompt string, reader *bufio.Reader, defaultInput string) string {
 	fmt.Print(prompt)
 	response, err := reader.ReadString('\n')
 	if err != nil {
 		panic(err)
 	}
 	fmt.Print("\n")
-	return strings.TrimRight(response, "\n")
+	input := strings.TrimRight(response, "\n")
+	if input != "" {
+		return input
+	} else {
+		return defaultInput
+	}
 }
 
-func promptForInput(prompt string) string {
-	return promptForInputWithPipes(prompt, bufio.NewReader(os.Stdin))
+func promptForInput(prompt string, defaultInput string) string {
```

Comment:
I'm not sure the value of adding a `defaultInput` to this generic `promptForInput`function? 

AFAICT there are no callers `promptForInput` that want to specify a default input, so they all end up having to add `""` to their argument list, which is weird

Reply:
Agreed, I'll move it down a level into `promptForInputWithPipes` and call that directly from `promptYesOrNo`

Reply:
I think that' still overcomplicating things. Why not do this logic in `promptYesOrNo` ? that's the only place that the default makes sense